test/src/system/custom_element_test.coffee

```diff
@@ -407,3 +407,18 @@ testGroup "<label> support", template: "editor_with_labels", ->
     label.click()
     assert.notEqual getEditorElement(), document.activeElement
     done()
+
+testGroup "default ARIA label", template: "editor_without_aria_label", ->
+  test "assigns aria-label to the text of the element's <labels>", (done) ->
+    assert.equal getEditorElement().getAttribute("aria-label"), "Label 1 Label 2 Label 3"
```

Comment:
These tests probably don't deserve their own templates, though I wasn't sure how to access different <trix-editor> elements within a single template.